README.md

```diff
@@ -1,8 +1,8 @@
-Logging to InsightOps in Ruby
+Logging to R7Insight in Ruby
```

Comment:
`R7Insight` doesn't look right for the non-code references here I think

Reply:
True, what would you prefer? `Rapid7 Insight Platform`?